index.html

```diff
@@ -0,0 +1,259 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="./styles.css" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>BountyDo</title>
+</head>
+
+<body>
+
+  <header>
+
+      <div class="wrapper__logo">
+          <small class="bounty">BountyDo</small>
+      </div>
+      <div class = "wrapper__auth">
+        <a class = "auth_buttons" href="logout.html">
+          <button class="button">Log Out</button>
+        </a>
+        <a class = "auth_buttons" href="index.html">
+          <button class="button">Account</button>
+        </a>
+      </div>
+
+
+  </header>
+
+  <div class="main"> 
+    <div class="main__wrapper"> 
+      <div class="counter__container"> 
+        <div class="counter"> 
+          <h2 class="counter__title">Counter</h2> 
+          <div class="wrapper">
+            <div class="wrapper__counter">
+              <p class="wrapper__counter_completed">Completed tasks:</p>
+              <div class="output__field"></div>
+            </div>
+            <div class="wrapper__counter">
+              <p class="wrapper__counter_remaining">Remaining tasks:</p>
+              <div class="output__field"></div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="filter__container">
+        <div class="filter">
+          <h2 class="filter__title">Search</h2>
+          <div class="wrapper">
+            <div class="wrapper__filter">
+              <input type="text" class="input" name="input__filter" />
+              <button class="button">Search</button>
+            </div>
+            <h2 class="tagfilter__title">Tag filter</h2>
+            <div class="wrapper__filter">
+              <label class="radio__wrapper">
+                <input type="radio" name="work" />
+                Work
+              </label>
```

Comment:
http://htmlbook.ru/html/label . Use using `label` wrong Read this link and use introspection for solve this problrm